client/components/Main.js

```diff
@@ -1,4 +1,6 @@
 import React from 'react';
+import store from '../store';
+import { connect, Provider } from 'react-redux';
```

Comment:
No need to import Provider here

Reply:
Thanks Alejandra, I made your two changes. I also connected the Main component to the store using mapStateToProps and mapDispatchToProps. I'll test out a simple Users component on it while l wait for your response.